scripts/infra_ssh_config.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
```

Comment:
Can we add `set -e`? Had errors because I forgot to install this version of `audius-ctl`, would hate to think it might have populated the ssh config with garbage if not for the abort clause.